db/db_test.cc

```diff
@@ -1128,6 +1129,7 @@ TEST_F(DBTest, MetaDataTest) {
   Options options = CurrentOptions();
   options.create_if_missing = true;
   options.disable_auto_compactions = true;
+  options.enable_blob_files = true;
```

Comment:
I think this is also an artifact of the original design? I'd say let's revert this; this test case exercises some of the logic related to the old BlobDB and it would be confusing to enable the new implementation.